routes/users.js

```diff
@@ -112,7 +119,35 @@ usersRouter.get("/:userId/orders", isLoggedIn, isAdmin, async (req, res, next) =
         res.send(allOrdersByUser)
     } catch (error) {
         throw error;
-    };
+    }
+})
+
+//NEW ADMIN PATCH AND DELETE//
+usersRouter.patch('/:id', async (req, res, next)=>{
+    try{
+        const {id} = req.params;
+        const {isAdmin}= req.body;
```

Comment:
For isAdmin, we can pass it inside the parameters:

usersRouter.patch("/:id", isAdmin, async (req, res, next) =>